trendyqc/trend_monitoring/management/commands/utils/_multiqc.py

```
@@ -596,9 +597,11 @@ def import_instances(self):
                     instance_model_name = type(instance).__name__
                     msg = (
                         "Could not be imported because of "
-                        f"{instance_model_name}:\n{e}"
+                        f"`{instance_model_name}`:\n```{e}```"
                     )
                     self.messages.append((msg, "error"))
+                    logger = logging.getLogger(__name__)
+                    logger.error("Failed to import instance: %s", traceback.format_exc())
```

Ruff (0.8.0)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)